### PR TITLE
add section in privacy considerations about issuer

### LINKED ISSUE #209: Does pseudo-anonymity require the issuer to cooperate?
This question came up in our use cases call in issue https://github.com/w3c/vc-use-cases/issues/10

How explicit should we be in the DM about issuer’s role in creating ZKP credentials for things like withholding “name” as discussed in the use cases section 3.5. 
https://w3c.github.io/vc-use-cases/#professional-credentials -- see the Paula example in item C.5 Social authority.

in other words, can I say “I’m in control of credential X in which I’m the subject, but you can’t know who I really am”

### PR DESCRIPTION
**Pull request by brentzundel**
This PR fixes #209 by adding a section to privacy considerations about the issuers role in providing and not violating privacy protections.

Signed-off-by: Brent Zundel <brent.zundel@gmail.com>

***
<a href="https://pr-preview.s3.amazonaws.com/brentzundel/vc-data-model/pull/830.html" title="Last updated on Nov 22, 2021, 10:32 PM UTC (5b88bf3)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/830/f01cdd3...brentzundel:5b88bf3.html" title="Last updated on Nov 22, 2021, 10:32 PM UTC (5b88bf3)">Diff</a>

### CODE REVIEW

**Comment by dmitrizagidulin** [comment]
:+1:

File: index.html
Comment (TallTed) [inline index.html:4613]:
```suggestion
It cannot be overstated that <a>verifiable credentials</a> rely on a high
```

File: index.html
Comment (TallTed) [inline index.html:4624]:
```suggestion
It should also be noted that, in addition to a reliance on <a>issuer</a>
participation to provide <a>verifiable credential</a> capabilities that help
```

File: index.html
Comment (TallTed) [inline index.html:4627]:
```suggestion
<a>issuer</a> might sign <a>verifiable credentials</a> using a signature scheme
```

**Review by dlongley — APPROVED** [review]
Approving w/@TallTed's suggestions.

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2021-11-03-vcwg#section4-1) on 2021-11-03

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 4.1. add section in privacy considerations about issuer (pr vc-data-model#830)

_See github pull request [vc-data-model#830](https://github.com/w3c/vc-data-model/pull/830)._




**Brent Zundel:** 830: adds a privacy consideration with issuer participation related to the possible privacy protections. encourage folks to jump in and review.  

**Kyle Den Hartog:** no normative changes, makes sense.  

> *Manu Sporny:* agree, it's editorial..

**Kyle Den Hartog:** let's manage it as an editorial change into 1.1.  

**Brent Zundel:** any opposition to adding the v1.1 label?.  
… if we have a issue already tagged errata, we don't need to tag it as such. it's attached to 209.  

</details>

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2021-11-10-vcwg#section5-1) on 2021-11-10

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 5.1. Does pseudo-anonymity require the issuer to cooperate? (issue vc-data-model#209)

_See github issue [vc-data-model#209](https://github.com/w3c/vc-data-model/issues/209)._




> *Brent Zundel:* related to PR #830.

_See github pull request [vc-data-model#830](https://github.com/w3c/vc-data-model/pull/830)._




**Brent Zundel:** pr exists for this issue so it will automatically close when pr is merged.  

</details>

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2021-11-17-vcwg#section3-4) on 2021-11-17

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 3.4. add section in privacy considerations about issuer (pr vc-data-model#830)

_See github pull request [vc-data-model#830](https://github.com/w3c/vc-data-model/pull/830)._




**Brent Zundel:** I raised this PR to add a section to privacy considerations.  
… to call out coordination by the issuer to participate in the process of this.  
… it also points out that in some cases the issuer is not deliberately subverting the privacy aspects when they're participating.  
… please give it a review and I think this one is ready for CCG review.  

**Manu Sporny:** My concern is that editors aren't being tagged on these.  
… I was surprise that we're not being tagged on this.  
… I'm wondering if we're missing a codeowners file.  

**Kyle Den Hartog:** reiterates what Manu said.  

**Brent Zundel:** Editors should be notified any time new PRs are being raised.  

**David Chadwick:** going back to the previous PR there were a few mistakes for the intermediate credential.  
… suggests some changes so that the intermediate value is correct around the issuer values.  

**Manu Sporny:** let's open an issue on this. There might be an issue of the understanding of what should happen on this.  

**David Chadwick:** I think we should block merging the other until we can resolve this.  

**Brent Zundel:** Can you open an issue to track the thing that needs to be resolved for this.  

**Manu Sporny:** I'm on it.  

</details>

**Review by kdenhartog — APPROVED** [review]

File: index.html
Comment (msporny) [inline index.html:4614]:
```suggestion
degree of trust in <a>issuers</a>. The degree to which a <a>holder</a> might take
```

File: index.html
Comment (msporny) [inline index.html:4631]:
```suggestion
issued <a>credential</a>, it might be possible for the <a>issuer</a> to track
```

**Review by msporny — APPROVED** [review]
LGTM with the two very minor wording changes to avoid confusion between normative language and what was intended.

File: index.html
Comment (TallTed) [inline index.html:4620]:
```suggestion
incorporate them into the <a>verifiable credentials</a> they issue.
```

**Review by TallTed — CHANGES_REQUESTED** [review]
I concur with @msporny's tweaks, and add one more tiny one.

**Review by TallTed — APPROVED** [review]
Print it!

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2021-12-01-vcwg#section4-5) on 2021-12-01

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 4.5. Does pseudo-anonymity require the issuer to cooperate? (issue vc-data-model#209)

_See github issue [vc-data-model#209](https://github.com/w3c/vc-data-model/issues/209)._




_See github pull request [vc-data-model#830](https://github.com/w3c/vc-data-model/pull/830)._




**Brent Zundel:** I raised a PR to address this, got some feedback, that PR has existed for 21 days, but not labeled properly -- waiting for PR to formally be accepted..  
… Last thing I think Ted made, requested some changes, accepted your suggestions -- re-review would be good..  

**Ted Thibodeau Jr.:** I'll look at that..  

</details>

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**
